src/jdk.management/share/classes/com/sun/management/internal/GarbageCollectionNotifInfoCompositeData.java

```diff
@@ -65,19 +63,17 @@ public static CompositeData toCompositeData(GarbageCollectionNotificationInfo in
     }
 
     private CompositeType getCompositeTypeByBuilder() {
-        @SuppressWarnings("removal")
-        final GcInfoBuilder builder = AccessController.doPrivileged (new PrivilegedAction<GcInfoBuilder>() {
-                public GcInfoBuilder run() {
-                    try {
-                        Class<?> cl = Class.forName("com.sun.management.GcInfo");
-                        Field f = cl.getDeclaredField("builder");
-                        f.setAccessible(true);
-                        return (GcInfoBuilder)f.get(gcNotifInfo.getGcInfo());
-                    } catch(ClassNotFoundException | NoSuchFieldException | IllegalAccessException e) {
-                        return null;
-                    }
-                }
-            });
+
+        GcInfoBuilder builder = null;
+        try {
+            Class<?> cl = Class.forName("com.sun.management.GcInfo");
+            Field f = cl.getDeclaredField("builder");
+            f.setAccessible(true);
+            builder = (GcInfoBuilder) f.get(gcNotifInfo.getGcInfo());
```

Comment:
```suggestion
            builder = (GcInfoBuilder)f.get(gcNotifInfo.getGcInfo());
```

Reply:
OK, it was missing the space before, when moving it around I added the space, as that's what we normally have.  In this directory there is a mixture of the two styles, with and without spaces.  I hope it's OK to let this one migrate to the more modern style. 8-)